### PR TITLE
Add the laserdisc scala client

### PR DESCRIPTION
**Pull request by nashid**
Add laserdisc scala client - https://github.com/laserdisc-io/laserdisc

### CODE REVIEW

**Comment by barambani** [comment]
This will complete the work here https://github.com/laserdisc-io/laserdisc/issues/61 . Adding reference.

**Comment by barambani** [comment]
@antirez this is ready to be reviewed. Please let us know if the process for adding clients has changed or if we have to add more details to the PR. Thanks.

File: clients.json [hunk 1]
```diff
@@ -544,6 +544,16 @@
     "authors": ["vitaliykhamin"]
   },
 
+  {
+    "name": "laserdisc",
+    "language": "Scala",
+    "repository": "https://github.com/laserdisc-io/laserdisc",
+    "description": "Future free Fs2 native pure FP Redis client http://laserdisc.io",
+    "authors": ["sirocchj", "barambani"],
```
Comment (itamarhaber) [inline clients.json]: This should be a Twitter handle (not GitHub) and 'sirocchj' does not exist.

File: clients.json [hunk 2]
```diff
@@ -544,6 +544,16 @@
...
+    "recommended": true,
```
Comment (itamarhaber) [inline clients.json]: This property should not be a part of the PR - it is left to the maintainers' discretion based on community feedback.

File: clients.json (hunk 1 above)
Comment (nashid) [reply clients.json]: User `sirocchj` do exist https://github.com/sirocchj.

But 👍 for the pointer - added the twitter handle.

File: clients.json (hunk 2 above)
Comment (nashid) [reply clients.json]: comments addressed.

**Comment by nashid** [comment]
@antirez comments addressed - can you please have a look. 

If you need additional details do let me know.

**Comment by barambani** [comment]
@itamarhaber do you think there's any way to progress this or we should close it ? Thanks.

**Pull request merged**